spec/controllers/provider_foreman_controller_spec.rb

```diff
@@ -95,7 +95,7 @@
     end
 
     it "renders explorer based on RBAC access to feature 'provider_foreman_add_provider'" do
-      set_user_privileges user_with_feature %w(provider_foreman_add_provider)
+      login_as user_with_feature %w(provider_foreman_add_provider)
```

Comment:
In other terms, we should use `login_as` when you want a **real** user model, saved to the db, and `stub_user` when you want a user object with stubbed responses (`stub_user` currently creates a real user model but I'd soon like to try using an RSpec test double or a FactoryGirl object (build, not create))

This is a distinction I'd really like to push, as we can avoid people doing db writes for everything just for a sake of a user object when they can just stub it.